QueryEngine/CompilationOptions.h

```diff
@@ -19,7 +19,7 @@
 
 enum class ExecutorDeviceType { CPU, GPU };
 
-enum class ExecutorOptLevel { Default, LoopStrengthReduction, ReductionJIT };
+enum class ExecutorOptLevel { Default, O1, O2, O3, LoopStrengthReduction, ReductionJIT };
```

Comment:
Need a comment saying that O1, O2, O3 shouldn't move.
Or should assign values explicitly O1=1, O2=2, O3=3

Reply:
Done